custom/covid/management/commands/update_case_index_relationship.py

```diff
@@ -35,11 +35,13 @@ def case_block(case):
     ).as_xml()).decode('utf-8')
 
 
-def update_cases(domain):
+def update_cases(domain, username):
     accessor = CaseAccessors(domain)
     case_ids = accessor.get_case_ids_in_domain(CASE_TYPE)
     print(f"Found {len(case_ids)} {CASE_TYPE} cases in {domain}")
 
+    user_id = username_to_user_id(username)
```

Comment:
Nitpick: check for `None` - as is, if someone mistypes the username, this will return `None` and the script will run but ac as the system user.